src/wallet/rpcwallet.cpp

```diff
@@ -3266,7 +3269,7 @@ static UniValue bumpfee(const JSONRPCRequest& request)
                 },
                 RPCResult{
                     RPCResult::Type::OBJ, "", "", {
-                        {RPCResult::Type::STR, "psbt", "The base64-encoded unsigned PSBT of the new transaction. Only returned when wallet private keys are disabled."},
+                        {RPCResult::Type::STR, "psbt", "The base64-encoded unsigned PSBT of the new transaction." + std::string(want_psbt ? "" : " Only returned when wallet private keys are disabled. (DEPRECATED)")},
                         {RPCResult::Type::STR_HEX, "txid", "The id of the new transaction. Only returned when wallet private keys are enabled."},
```

Comment:
Needs to be hidden for psbtbumpfee.

Reply:
Oh.. I've grabbed your commit that adds `RPCResult::Type::HIDDEN` to handle this.